src/libs/antares/optimization-options/include/antares/optimization-options/options.h

```diff
@@ -28,16 +28,24 @@ struct SingleOptimOptions
 {
     std::string solverName = "sirius";
     std::string solverParameters;
+
+    // Reusing basis of first optimization (in case we have 2 weekly
+    // linear optimizations [not MILP])
+    bool solverAcquiresBasis = false;
+    bool solverExportsBasis = false;
```

Comment:
Here we add these 2 data members to `struct SingleOptimOptions`.
But `SingleOptimOptions` can be a quadratic optimization, and in this case, these members don't make sense (and stay unused).
This was done to make things work.

Instead, we should define 3 struct / classes : **SingleOptimOptions**, **LinearOptimOptions** and **QuadraticOptimOptions**, the last two inheriting from the first, and then pass the right one where needed (for this, see calls to functions **runWeeklyOptimization** and **OPT_PilotageOptimisationQuadratique**).